login.py

```diff
@@ -1,23 +1,46 @@
 from tkinter import *
-import Handlers
+import tkinter.messagebox
+
+#import Handlers
 
 wnd = Tk()
 
+def SignInCallBack():
+      
+      username1 = usernamevalue.get()
+      password1 = passwordvalue.get()
+
+      if len(username1) <= 0:
+          tkinter.messagebox.showwarning( "Message", "User Name can not be empty")
+      elif len(password1) <= 0:
+          tkinter.messagebox.showwarning( "Message", "password can not be empty")
+      else :
+          tkinter.messagebox.showwarning( "Message", "Not implemented yet..")
+

```

Comment:
Better if moved to a different module. This is our validation logic and shouldn't be mixed up with the UI logic.